pkg/gatherers/conditional/conditional_gatherer.go

```diff
@@ -286,13 +286,16 @@ func (g *Gatherer) updateAlertsCache(ctx context.Context, metricsClient rest.Int
 }
 
 func (g *Gatherer) updateVersionCache(ctx context.Context, configClient configv1client.ConfigV1Interface) error {
+	const logPrefix = "conditional gatherer: "
```

Comment:
This repeats in more methods here AFAIK so perhaps it would be better to create global constant.

Reply:
I realized that it doesn't make much sense. It should be either all conditional gatherer logs are prefix or none at all and in this particular case we already have file name which is `conditional_gatherer.go` added before the logs.